installer/prepare.sh

```diff
@@ -198,9 +198,15 @@ compile() {
         shift 1
     fi
     shift 2
+    local pkgs=""
```

Comment:
Set pkgs to "$*" and prefix in the debian branch?

Alternatively, I think you could just `debian=gcc, debian=libc6-dev` and be done with it.